recipe/bld.bat

```diff
@@ -0,0 +1,11 @@
+setlocal EnableDelayedExpansion
+
+:: Copy the [de]activate scripts to %LIBRARY_PREFIX%\etc\conda\[de]activate.d.
+:: This will allow them to be run on environment activation.
+FOR %%F IN (activate deactivate) DO (
+    IF NOT EXIST %LIBRARY_PREFIX%\etc\conda\%%F.d MKDIR %LIBRARY_PREFIX%\etc\conda\%%F.d
+    COPY %RECIPE_DIR%/%%F.bat %LIBRARY_PREFIX%\etc\conda\%%F.d\keras_%%F.bat
```

Comment:
That's actually not wrong.  That activate call should then call your keras activate script.  The script you're seeing there is conda's activate script.